sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/auth/SdaAccessDecisionManager.java

```diff
@@ -8,19 +8,36 @@
 package org.sdase.commons.spring.boot.web.auth;
 
 import java.util.List;
+import java.util.function.Supplier;
 import org.sdase.commons.spring.boot.web.auth.management.ManagementAccessDecisionVoter;
 import org.sdase.commons.spring.boot.web.auth.opa.OpaAccessDecisionVoter;
 import org.sdase.commons.spring.boot.web.auth.opa.OpaExcludesDecisionVoter;
 import org.springframework.security.access.vote.UnanimousBased;
+import org.springframework.security.authorization.AuthorizationDecision;
+import org.springframework.security.authorization.AuthorizationManager;
+import org.springframework.security.authorization.AuthorizationManagers;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.FilterInvocation;
 import org.springframework.stereotype.Component;
 
 @Component
-public class SdaAccessDecisionManager extends UnanimousBased {
+public class SdaAccessDecisionManager implements AuthorizationManager<FilterInvocation> {
+  private AuthorizationManager<FilterInvocation> authorizationManager;
 
   public SdaAccessDecisionManager(
       ManagementAccessDecisionVoter managementAccessDecisionVoter,
       OpaExcludesDecisionVoter opaExcludesDecisionVoter,
       OpaAccessDecisionVoter opaAccessDecisionVoter) {
-    super(List.of(managementAccessDecisionVoter, opaExcludesDecisionVoter, opaAccessDecisionVoter));
+    authorizationManager = AuthorizationManagers.allOf(managementAccessDecisionVoter, opaExcludesDecisionVoter, opaAccessDecisionVoter);
```

Comment:
Imho this must be `anyOf(…)`. In this previous implementation we always used `ACCESS_ABSTAIN` instead of deny. It means, that the voter returning it, does not decide for this request. The aggregation rule to allow a request was „_at minimum one granted and no deny decision, ignoring any abstain decisions_“. As we did not use deny explicitly, a request was denied, when all voters returned abstain.